### PR TITLE
add release-1.9 triggers to other repos

### PR DESCRIPTION
**Pull request by tuminoid**
Add release-1.9 triggers to dev-env, ipa-downloader and mariadb-image.

Update README.md for the new triggers.

Update milestone applier config to use 1.9 branches and milestones correctly.

### CODE REVIEW

**Comment by tuminoid** [comment]
/hold
Even though IPAM could use this right now for the milestone applier config, let's not update it until all content in CAPM3 is in, as those PRs would be labelled wrong then.

/override metal3-ubuntu-e2e-integration-test-main 
/cc @kashifest @Sunnatillo @Rozzii

File: prow/config/jobs/metal3-io/ironic-ipa-downloader.yaml [hunk 1]
```diff
@@ -33,6 +33,10 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: false
+  - name: metal3-centos-e2e-integration-test-release-1-9
```
Comment (kashifest) [inline prow/config/jobs/metal3-io/ironic-ipa-downloader.yaml:36]: shall we also remove the 1-6 triggers?

Comment (tuminoid) [reply prow/config/jobs/metal3-io/ironic-ipa-downloader.yaml:36]: Removal in another PR.

Comment (tuminoid) [reply prow/config/jobs/metal3-io/ironic-ipa-downloader.yaml:36]: We have agreed to follow best practice of not mixing adding and removing.

**Review by Sunnatillo — APPROVED** [review]
/lgtm

**Review by Rozzii — COMMENTED** [review]
/lgtm

**Pull request merged**